pyaedt/edb_core/padstack.py

```diff
@@ -551,3 +584,92 @@ def remove_pads_from_padstack(self, padstack_name, layer_name=None):
         self.padstacks[padstack_name].edb_padstack.SetData(newPadstackDefinitionData)
         self.update_padstacks()
         return True
+
+
+class EdbPadstackInstances:
+    """Manages EDB functionalities for padstack instances.
+
+    Examples
+    --------
+    >>> from pyaedt import Edb
+    >>> edbapp = Edb("myaedbfolder", edbversion="2021.2")
+    >>> edb_padstacks = edbapp.core_padstack_instance
+    """
+
+    def __init__(self, p_edb):
+        self._pedb = p_edb
+        self._padstack_instances = []
+
+    @property
+    def _builder(self):
+        """ """
+        return self._pedb.builder
+
+    @property
+    def _edb(self):
+        """ """
+        return self._pedb.edb
+
+    @property
+    def _edb_value(self):
+        """ """
+        return self._pedb.edb_value
+
+    @property
+    def _active_layout(self):
+        """ """
+        return self._pedb.active_layout
+
+    @property
+    def db(self):
+        """Db object."""
+        return self._pedb.db
+
+    @property
+    def _logger(self):
+        """ """
+        return self._pedb.logger
+
+    @property
+    def _layers(self):
+        """ """
+        return self._pedb.core_stackup.stackup_layers
```

Comment:
@ring630 Is this property  and the one above protected on purpose or shall they become public and in that case it would become something like:
```suggestion
    def layers(self):
        """ """
        return self._pedb.core_stackup.stackup_layers
```